scripts/uploadThemes.js

```diff
@@ -0,0 +1,119 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const childProcess = require('child_process')
+const commander = require('commander')
+const crypto = require('crypto')
+const fs = require('fs')
+const mkdirp = require('mkdirp')
+const path = require('path')
+
+process.on('uncaughtException', (err) => {
+  console.error('Caught exception:', err)
+  process.exit(1)
+})
+
+process.on('unhandledRejection', (err) => {
+  console.error('Unhandled rejection:', err)
+  process.exit(1)
+})
+
+function createManifestForUpload (id, name, crxFileHash) {
+  const manifestDir = path.join('build', 'themes', 'stable', 'extensions')
+  const manifestFile = path.join(manifestDir, 'extensionManifest.json')
+
+  const content =
+        [
+          [
+            id,
+            commander.setVersion,
+            crxFileHash,
+            name
+          ]
+        ]
+
+  mkdirp.sync(manifestDir)
+
+  fs.writeFileSync(manifestFile, JSON.stringify(content, null, 2))
+}
+
+function parseManifest (manifestFile) {
+  // Strip comments from manifest.json before parsing
+  const json = fs.readFileSync(manifestFile).toString('utf-8').replace(/\/\/#.*/g, '')
+  return JSON.parse(json)
+}
+
+function generateSHA256Hash (data) {
```

Comment:
For this and others, pls create another js file in something like `scripts/lib/util.js`
And then you can 
`const {generateSHA256Hash, generateSHA256HashOfFile} = require('./lib/util')`